nilearn/decoding/decoder.py

```diff
@@ -1065,6 +1065,9 @@ def _predict_dummy(self, n_samples):
             )
         return scores.ravel() if scores.shape[1] == 1 else scores
 
+    def _more_tags(self):
+        return {"require_y": True}
```

Comment:
You can easily cover this line by adding a check that `require_y` is set to True and `mulitoutput` is False for the classifier and True for the regressor. It allows to check that the MRO works as expected as well.